sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkExecutionOptions.java

```diff
@@ -92,13 +94,35 @@ CosmosBulkExecutionOptions setMaxMicroBatchSize(int maxMicroBatchSize) {
 
     /**
      * The maximum concurrency for executing requests for a partition key range.
+     * By default, the maxMicroBatchConcurrency is 1.
      *
      * @return max micro batch concurrency
      */
-    int getMaxMicroBatchConcurrency() {
+    public int getMaxMicroBatchConcurrency() {
         return maxMicroBatchConcurrency;
     }
 
+    /**
+     * Set the maximum concurrency for executing requests for a partition key range.
+     * By default, the maxMicroBatchConcurrency is 1.
+     * It only allows values &ge;1 and &le;5.
+     *
+     * Attention! Please adjust this value with caution.
+     * By increasing this value, more concurrent requests will be allowed to be sent to the server,
+     * in which case may cause 429 or request timed out due to saturate local resources, which could degrade the performance.
+     *
+     * @param maxMicroBatchConcurrency the micro batch concurrency.
+     *
+     * @return the bulk processing options.
+     */
+    public CosmosBulkExecutionOptions setMaxMicroBatchConcurrency(int maxMicroBatchConcurrency) {
```

Comment:
should we have beta tag for these? or are these APIs stable enough?

Reply:
IMO Stable enough - Annie has restricted the values so customers can't shoot themselves. And lifting the constraints would not be breaking. Naming is hopefully something we wouldn't have to churn on.